interfaces/cython/cantera/composite.py

```diff
@@ -649,6 +649,10 @@ def append(self, state=None, **kwargs):
           the full-state setters::
 
               mystates.append(T=300, P=101325, X={'O2':1.0, 'N2':3.76})
+
+        By default, the mass or mole fractions will be normalized i.e negative values
+        are trumncated and the mass or mole fractions are sum up to 1.0. If this
```

Comment:
Sorry, just noticed another typo here:
```suggestion
        are truncated and the mass or mole fractions sum up to 1.0. If this
```